packages/transformers/typescript-types/src/utils.js

```diff
@@ -26,3 +27,22 @@ export function isDeclaration(ts: TypeScriptModule, node: any): boolean {
     ts.isTypeAliasDeclaration(node)
   );
 }
+
+export function createImportSpecifier(
+  ts: TypeScriptModule,
+  propertyName: Identifier | void,
+  name: Identifier,
+  isTypeOnly: boolean = false,
+): ImportSpecifier {
+  const [majorVersion, minorVersion] = ts.versionMajorMinor
+    .split('.')
+    .map(num => parseInt(num, 10));
+  // The signature of createImportSpecifier had a breaking change in Typescript 4.5.
+  // see: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-5.html#type-modifiers-on-import-names
```

Comment:
Wowwwwww... TS doesn't follow semver?? Maybe we should pin our dependency on it?

Reply:
The other side of that coin: https://github.com/parcel-bundler/parcel/issues/5978